.travis.yml

```diff
@@ -135,7 +135,7 @@ script:
 after_script:
   - set -e
   - bash etc/ci-gather-coverage.sh
-  - bash <(curl -s https://codecov.io/bash)
+  - bash <(curl -s https://codecov.io/bash) -f "!./pkg/*" -f "!./extern/*" -f "!./build/*"
```

Comment:
Ahh, I bet I forgot to exclude `build` in past attempts, too, which might explain some things.